tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2465,15 +2468,17 @@ declare namespace ts {
         InElementType = 2097152,
         InFirstTypeArgument = 4194304,
         InTypeAlias = 8388608,
+        WriteComputedProps = 1073741824,
         /** @deprecated */ WriteOwnNameForAnyLike = 0,
-        NodeBuilderFlagsMask = 848330091
+        NodeBuilderFlagsMask = 1922071915
     }
     export enum SymbolFormatFlags {
         None = 0,
         WriteTypeParametersOrArguments = 1,
         UseOnlyExternalAliasing = 2,
         AllowAnyNodeKind = 4,
         UseAliasDefinedOutsideCurrentScope = 8,
+        WriteComputedProps = 16,
```

Comment:
Should we mark this new flag as `@internal`, or do we want to make using it part of the public API?